libs/styles/consonant-play-button.css

```diff
@@ -51,14 +51,16 @@
 }
 
 .modal-img-link .consonant-play-btn:hover {
-  background-color: var(--color-accent-down);
+  background-color: rgb(0 0 0 / 100%);
 }
 
 .modal-img-link .consonant-play-btn:focus,
 .modal-img-link .consonant-play-btn:focus-visible {
-  outline: var(--play-border-width-m) solid var(--link-color-dark);
+  outline: var(--play-border-width-l) solid var(--color-black);
```

Comment:
Since just the size is changing, could we not declare just `outline-width`?

Reply:
The color changed too. From blue to black.

Reply:
What I mean by this is that we declare color and type on L41. This declaration just updates width. Could we not declare just the width here?

Reply:
No. This black outline is added on the outside of the focus ring so the focus ring has much better accessibility contrast (this was suggested by Taryn). Removing the color from this line removes that black that creates this effect
<img width="180" alt="image" src="https://github.com/user-attachments/assets/2d7568a0-3b32-4cd0-b148-e807fbd17dd6">

Reply:
It's not all that important, but here's the change I was suggesting:
```suggestion
  outline-width: var(--play-border-width-l);
```
On L41, you already have `outline: 0 solid var(--color-black);`, so the style and color are set. You're only updating the width, so unless there are other overrides, you just need to re-declare the width here. Made those changes locally and the styles are maintained:
![Screenshot 2024-11-25 at 12 22 36](https://github.com/user-attachments/assets/4c3d55d1-33a5-4881-a0d7-ac1f99bea73d)